tests/mtx/message.test.ts

```diff
@@ -1311,20 +1328,37 @@ const uplinkMessages: TMessageList = [
         destination: 0xaaaa
     },
     {
-        name: 'getEnergyDay',
+        name: 'getDayDemand',
```

Comment:
it would be nice to keep sorting by name

Reply:
fixed